doc/specs/stdlib_sorting.md

```diff
@@ -496,107 +559,126 @@ of size `16**3`, with characters drawn from the set "a"-"p":
   random order.
 
 These benchmarks have been performed on two different compilers, both
-on a MacBook Pro, featuring a 2.3 GHz Quad-Core Intel Core i5, with 8
-GB 2133 MHz LPDDR3 memory. The first compiler was GNU Fortran
-(GCC) 10.2.0, with the following results:
+on WSL with Ubuntu-20.04, Intel(R) Core(TM) i7-10700 CPU @ 2.9GHz, with
+32 GB DDR4 memory. The first compiler is GNU Fortran (GCC) 9.4.0, with
```

Comment:
It might be good to specify which compiler options/optimizations were used (if non-default ones were used)

Reply:
> It might be good to specify which compiler options/optimizations were used (if non-default ones were used)

It is built with default CMakeLists.txt